homeassistant/components/fritz/const.py

```diff
@@ -0,0 +1,30 @@
+"""Constants for the FRITZ!Box Tools integration."""
+
+DOMAIN = "fritz"
+DOMAIN_FRITZ = "fritz"
+
+SUPPORTED_DOMAINS = ["device_tracker"]
+
+ATTR_HOST = "host"
+
+DEFAULT_DEVICE_NAME = "Unknown device"
+DEFAULT_HOST = "192.168.178.1"
+DEFAULT_PORT = 49000
+DEFAULT_USERNAME = ""
+
+DEFAULT_USE_TRACKER = True
+DEFAULT_USE_WIFI = True
+DEFAULT_USE_PORT = True
+DEFAULT_USE_DEFLECTIONS = True
+DEFAULT_USE_PROFILES = True
+
+DEFAULT_PROFILES = None
+
+SERVICE_RECONNECT = "reconnect"
+SERVICE_REBOOT = "reboot"
+
+ERROR_CONNECTION_ERROR = "connection_error"
+
+TRACKER_SCAN_INTERVAL = 30
+SWITCH_SCAN_INTERVAL_1 = 30
```

Comment:
remve unneeded stuff here

Reply:
Not only in this line but in the whole of `const.py`

Reply:
Cleanup unused vars